locales/pt/json.json

```diff
@@ -307,16 +306,14 @@
     "Guinea-Bissau": "Guiné-Bissau",
     "Guyana": "Guiana",
     "Haiti": "Haiti",
-    "Having second thoughts about cancelling your subscription? You can instantly reactivate your subscription at any time until the end of your current billing cycle. After your current billing cycle ends, you may choose an entirely new subscription plan.": "Having second thoughts about cancelling your subscription? You can instantly reactivate your subscription at any time until the end of your current billing cycle. After your current billing cycle ends, you may choose an entirely new subscription plan.",
-    "Having second thoughts about cancelling your subscription? You can instantly reactive your subscription at any time until the end of your current billing cycle. After your current billing cycle ends, you may choose an entirely new subscription plan.": "Está a pensar em cancelar a sua assinatura? Pode reactivar instantaneamente a sua subscrição a qualquer momento até ao fim do seu ciclo de facturação actual. Após o fim do ciclo, pode escolher uma subscrição diferente.",
+    "Having second thoughts about cancelling your subscription? You can instantly reactivate your subscription at any time until the end of your current billing cycle. After your current billing cycle ends, you may choose an entirely new subscription plan.": "Está com dúvidas sobre o cancelamento da sua subscrição? Pode reativar instantaneamente a sua subscrição a qualquer momento até ao final do seu ciclo de faturação atual. Após o término do seu ciclo de faturação atual, poderá escolher um plano de subscrição totalmente novo.",
+    "Having second thoughts about cancelling your subscription? You can instantly reactive your subscription at any time until the end of your current billing cycle. After your current billing cycle ends, you may choose an entirely new subscription plan.": "Está com dúvidas sobre o cancelamento da sua subscrição? Pode reativar instantaneamente a sua subscrição a qualquer momento até ao final do seu ciclo de faturação atual. Após o término do seu ciclo de faturação atual, poderá escolher um plano de subscrição totalmente novo.",
     "Heard Island & Mcdonald Islands": "Ilhas Heard e McDonald",
     "Heard Island and McDonald Islands": "Ilha Heard e Ilhas McDonald",
     "Hello!": "Olá!",
     "Hide Content": "Esconder O Conteúdo",
     "Hold Up!": "Espera!",
     "Holy See (Vatican City State)": "Vatican City",
-    "Honduras": "Honduras",
```

Comment:
Keys do not need to be deleted. When translations match, they are simply specified in the "excludes" file.